docassemble/EFSPIntegration/data/questions/any_filing_interview.yml

```diff
@@ -245,15 +248,15 @@ fields:
   - no label: review_fees
     datatype: checkboxes
     choices:
-      - I agree to pay the listed fees of ${ currency(fee_total(fees_resp)) }: agrees_to_pay_fees
+      - ${ 'I agree to pay the listed fees of' + str(currency(fee_total(fees_resp))) if has_fees else 'No fees' }: agrees_to_pay_fees
```

Comment:
Just flagging that we'll want to change this to a template/conditional question for translation purposes at some point. not urgent though